### ISSUE TITLE
ScrollView is still scrollable when dragging delegates

### ISSUE THREAD

**Issue by Cuteivist** [status-im/status-desktop#10897]
# Bug Report

## Description

It is possible to scroll StatusScrollView when you drag StatusListItem. Maybe issue is also happening for other controls.

## Steps to reproduce

1. Go to settings -> Profile
2. Drag vertically on any control. Scroll view will move few pixels.

#### Expected behavior

Scroll View is consistent everywhere and is not possible to scroll it by drag.

#### Actual behavior

https://github.com/status-im/status-desktop/assets/11396062/c7a52878-76ed-40e3-a76c-e9a1837f79ba

https://github.com/status-im/status-desktop/assets/11396062/688530d3-752b-43a3-b98a-262f6645807e

### Additional Information

- Status desktop version:  master
- Operating System: Windows and Mac

**Comment by Cuteivist** [comment]
@igor-sirotin

**Comment by igor-sirotin** [comment]
Indeed, there's a problem with this. But it seems to be a Qt bug which we can't easily workaround.

`ScrollView` is excepted not to be "draggable by mouse" (though it should still process real touch screen events). The fact that it's stealing mouse for a moment is a bug that happens when a mouse event takes place in a child of `contentItem`.

Reference Qt bugs:
- https://bugreports.qt.io/browse/QTBUG-106489
- https://bugreports.qt.io/browse/QTBUG-79651

### Potential workaround

I found a workaround, but the scrolling becomes extremely uncomfortable. Set `synchronousDrag: false` here:
https://github.com/status-im/status-desktop/blob/2a253be291d4c2e8fa86ca93b142edeb3dc08cd4/ui/StatusQ/src/StatusQ/Core/StatusScrollView.qml#L169

### What are we gonna do?

IMO, this is not a terrible issue, I could live with it. 
Though I agree that it looks like it's intended to be draggable. And therefore looks like a bug.

@Cuteivist @jrainville @caybro wdyt?

**Comment by Cuteivist** [comment]
> What are we gonna do?
IMO, this is not a terrible issue, I could live with it.
Though I agree that it looks like it's intended to be draggable. And therefore looks like a bug.

I agree, let's park it for now. We can see in the future whether this issue is noticeable at all for the users.

**Comment by iurimatias** [comment]
we might not want to do this? cc: @John-44 @benjthayer

**Comment by benjthayer** [comment]
> we might not want to do this? cc: @John-44 @benjthayer

Agree it's not ideal to have scroll on drag randomly in some areas of the app but its perhaps not the highest priority, given it will likely only be discovered by accident and does not have a big impact on the usability of the app. WDYT @John-44 ?

**Comment by noeliaSD** [comment]
Moved to next milestone due to team capacity and priorities.

**Comment by jrainville** [comment]
Closing as stale
